doc/getting-started.adoc

```diff
@@ -103,7 +110,7 @@ Assuming your `NIX_PATH` includes `mobile-nixos=/path/to/mobile-nixos` you can
 import the Mobile NixOS configuration for your device by doing the following.
 
 ```nix
-# configuration.nix
+# configuration.nix (on your actual mobile device)
```

Comment:
the thing is, there is no `configuration.nix` :grimacing: 

ref: https://github.com/NixOS/mobile-nixos/issues/338